canvas-gamification.iml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you accidentally committed this file. Please remove it if that's the case.